app/src/main/java/pub/devrel/easypermissions/sample/MainActivity.java

```diff
@@ -58,6 +59,18 @@ public void onClick(View v) {
         });
     }
 
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if (requestCode == EasyPermissions.SETTINGS_REQ_CODE) {
```

Comment:
Since we're not passing a DialogListener in this example let's just drop this block too.  This can be the 'developer does not care about the result of the settings request' example and the other file can show the more in-depth options.